app/models/ability.rb

```diff
@@ -0,0 +1,14 @@
+class Ability
+  include CanCan::Ability
+
+  def initialize(user)
+    user ||= User.new # Guest user
+
+    if user.admin?
+      can :manage, :all
+    else
+      can :read, Post
```

Comment:
- [ ] Awesome! :clap: However, I think the requirements asked that users have right and privileges to delete/destroy posts/commtns if they want to but it seems you only allow users the `Read` functionality of the `CRUD`. Kindly update this to ensure authors can delete posts if they so will.
- Example:
```
can :manage, Post, author: user  # owner of post can perform all view, update or destroy own post if Post.author == user
can :manage, Comment, author: user
can :read, :all
```

Reply:
well resolved! ✔️